### PR TITLE
Spring Boot Database: MariaDB

### LINKED ISSUE #420: Spring Boot Database: MariaDB
Similar to MySQL

Maybe it should be done after the refactoring : https://github.com/jhipster/jhipster-lite/issues/419

### PR DESCRIPTION
**Pull request by seraphinandrieux**
- fix #420

### CODE REVIEW

**Comment by pascalgrimaud** [comment]
Thanks @seraphinandrieux 
I'll review it soon

**Comment by pascalgrimaud** [comment]
Another comment: can you add MariaDB to the CI plz ?
- need a new entry in GitHub Actions
- a new Json file
- a new test in generate.sh

File: src/main/java/tech/jhipster/lite/generator/server/springboot/database/mariadb/domain/MariaDBService.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  void addSpringDataJpa(Project project);
+  void addMariaDBDriver(Project project);
+  void addHikari(Project project);
+  void addHibernateCore(Project project);
+  void addDockerCompose(Project project);
+  void addJavaFiles(Project project);
+  void addProperties(Project project);
+  void addLoggerInConfiguration(Project project);
+  void addTestcontainers(Project project);
```
[the remark this answers is not in the document]
Comment (seraphinandrieux) [reply src/main/java/tech/jhipster/lite/generator/server/springboot/database/mariadb/domain/MariaDBService.java:16]: I created an issue to do it for other DB => https://github.com/jhipster/jhipster-lite/issues/901

**Review by pascalgrimaud — APPROVED** [review]

**Comment by seraphinandrieux** [comment]
$100 bounty claimed : https://opencollective.com/generator-jhipster/expenses/64890

**Comment by pascalgrimaud** [comment]
@seraphinandrieux : approved

**Pull request merged**